src/source/errorSchema.json

```diff
@@ -1,22 +1,42 @@
 {
-    "oneOf": [
-        {
-            "type": "object",
-            "properties": {
-                "code": {
-                    "type": "number"
+    "errorSchemaSDK": {
+        "oneOf": [
+            {
+                "type": "object",
+                "properties": {
+                    "code": {
```

Comment:
Can we keep the same what is present in open rpc
{
						"type": "object",
						"additionalProperties": false,
						"required": [
							"code",
							"message"
						],
						"properties": {
							"code": {
								"title": "errorObjectCode",
								"description": "A Number that indicates the error type that occurred. This MUST be an integer. The error codes from and including -32768 to -32000 are reserved for pre-defined errors. These pre-defined errors SHOULD be assumed to be returned from any JSON-RPC api.",
								"type": "integer"
							},
							"message": {
								"title": "errorObjectMessage",
								"description": "A String providing a short description of the error. The message SHOULD be limited to a concise single sentence.",
								"type": "string"
							},
							"data": {
								"title": "errorObjectData",
								"description": "A Primitive or Structured value that contains additional information about the error. This may be omitted. The value of this member is defined by the Server (e.g. detailed error information, nested errors etc.)."
							}
						}
					}

Reply:
full errorschema from open rpc must be added here including the description

Reply:
search for "x-error" in  https://rdkcentral.github.io/firebolt/requirements/latest/specifications/firebolt-open-rpc.json

Reply:
addressed the comment